src/DIRAC/RequestManagementSystem/Agent/CleanReqDBAgent.py

```diff
@@ -152,8 +164,28 @@ def execute(self):
                     continue
                 deleted += 1
 
+        # optional: Set Scheduled requests to Cancelled if older than threshold
+        if self.cancelGraceDays > 0:
+            statusList = ["Scheduled"]
+            result = self.requestClient().getRequestIDsList(statusList, self.DEL_LIMIT)
+            if not result["OK"]:
+                self.log.error("Failed to get list of Scheduled requests: %s" % result["Message"])
+                return result
+            requestIDsList = result["Value"]
+            cancelled = 0
+            for requestID, status, lastUpdate in requestIDsList:
+                if lastUpdate < cancelTime:
+                    self.log.info("Cancelling overdue request '%s'" % (requestID))
+                    cancelReq = self.requestClient().cancelRequest(requestID)
+                    if not cancelReq["OK"]:
+                        self.log.error("Unable to cancel request '%s': %s" % (requestID, cancelReq["Message"]))
+                        continue
+                    cancelled += 1
+
         gMonitor.addMark("KickedRequests", kicked)
         gMonitor.addMark("DeletedRequests", deleted)
+        gMonitor.addMark("CancelledRequests", cancelled)
         self.log.info("execute: kicked assigned requests = %s" % kicked)
         self.log.info("execute: deleted finished requests = %s" % deleted)
+        self.log.info("execute: cancelled overdue requests = %s" % cancelled)
```

Comment:
```suggestion
        self.log.info("execute: cancelled overdue requests", str(cancelled))
```
and please fix similarly the 2 previous lines too.